MapboxNavigationTests/ImageDownloaderTests.swift

```diff
@@ -12,7 +12,7 @@ class ImageDownloaderTests: XCTestCase {
 
     var downloader: ReentrantImageDownloader?
 
-    let imageURL = URL(string: "https://zombo.com/lulz/selfie.png")!
+    let imageURL = URL(string: "https://github.com/mapbox/mapbox-navigation-ios/blob/master/docs/img/navigation.png")!
```

Comment:
Wow, haha. The test doesn’t actually download this image from the network, right?

Reply:
it is a live test, added in #1046, selfie.png disappeared from the interwebs earlier today. w/i the repo sounds more reasonable but I'm up for stubbing the request now when we've linked OHHTTPStubs.

Reply:
@Udumft can we implement this suggestion? Live tests aren't the best idea.

Reply:
I tried this test with invalid image, I tried it with no internet connection and it works every time.
Maybe I misunderstood the point of the test, but since `downloader` is configured with `ImageLoadingURLProtocolSpy` like this: https://github.com/mapbox/mapbox-navigation-ios/blob/6d10ac02bc7ecb038fa770d2d5fad88bf651ad87/MapboxNavigationTests/ImageDownloaderTests.swift#L21-L24
it will always "download" an image. At the same time not configuring the downloader will make `ImageLoadingURLProtocolSpy` checks fail.
Maybe we should re-think these tests, or I missed something?